### PR TITLE
Improve performance of CI

### PR DESCRIPTION
**Pull request by Recmo**
## CI

Starting performance: 22 min 20 sec (7 min 27 sec till test starts)

## Local

Non-instrumented: 1:27.20 (was 2:57.08)

Instrumented: (was 11:07.10)

### CODE REVIEW

File: truffle.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+const TestRPC = require("ethereumjs-testrpc");
 
 module.exports = {
   networks: {
     development: {
-      host: "localhost",
-      port: 8545,
-      network_id: "*" // Match any network id
+      network_id: "*",
+      provider: TestRPC.provider()
```
Comment (Recmo) [inline truffle.js]: This helps a lot. Eliminates all (slow) http traffic during test. Unfortunately does not get used when runing solcover.

File: test/AccessControl.js [hunk 2]
```diff
@@ -3,24 +3,32 @@ import EvmError from "./helpers/EVMThrow";
...
       accessControl.address
     );
 
     exampleRole = roles.example;
+    snapshot = await saveBlockchain();
+  });
+
+  beforeEach(async () => {
+    await restoreBlockchain(snapshot);
+    snapshot = await saveBlockchain();
```
Comment (krzkaczor) [inline test/AccessControl.js]: Is this because you cannot restore twice the same snapshot? weeeiiirrddd

File: test/helpers/evmCommands.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+export const promisify = func => async (...args) =>
+  new Promise((accept, reject) =>
+    func(...args, (error, result) => (error ? reject(error) : accept(result)))
+  );
+
+export const rpcCommand = method => async (...params) =>
+  (await promisify(web3.currentProvider.sendAsync)({
```
Comment (krzkaczor) [inline test/helpers/evmCommands.js]: why did you decided to use async?

File: test/AccessControl.js (hunk 2 above)
Comment (Recmo) [reply test/AccessControl.js]: Yeah, seems so. I think it's more of a rollback operation than a restore.

**Pull request merged**